CONTRIBUTING.rst

```diff
@@ -18,12 +18,72 @@ first in an appropriate venue, drafting a PEP and gathering feedback, and
+
+* **Rejected**, **Deferred**, **Withdrawn** and **Superceded** PEPs should
+  usually only be modified when necessary to correct technical issues with
+  header processing or that prevent content from rendering as originally
+  intended. Occasionally, a Deferred or even Withdrawn PEP may be resurrected
+  with major updates, but it is often better to just propose a new one.
+
+* **Final** PEPs are considered complete; while small fixes to clearly
+  unintentional errors in the document that meaningfully impair understanding
+  may be considered, substantive changes to the content generally will not.
+
+* **Accepted** PEPs may be updated to reflect minor details of the final
+  implementation, but significant changes to the normative specification
+  should be avoided.
+
+* **Provisional** PEPs can be incrementally modified if needed, though
+  major rewrites and compatibility breakage are discouraged unless absolutely
+  necessary and any changes should be preceded by discussion and general
+  consensus among the authors and the relevant community.
+
+* **Active** PEPs can and should be updated whenever the information described
+  changes. Substantive changes to governance PEPs should be reviewed by the
+  Steering Council (by opening an
+  `issue <https://github.com/python/steering-council/issues>`__ in the SC repo)
+  and are subject to any further conditions stated in the PEP itself.
+
+* **Draft** PEPs are freely open for discussion and proposed modification
+  (though once submitted to the Steering Council or PEP-Delegate for final
+  review and resolution, significant alterations should be avoided).
+  Copyediting, formatting and proofreading can be opened here as pull requests,
+  while more substantive content changes should generally be first proposed
+  on the PEP's discussion thread listed in its ``Discussions-To`` header.
+
+Also, while fixing spelling and formatting issues as part of more substantive
+copyediting and proofreading of draft and active PEPs is encouraged,
+we generally advise against PRs that simply mass-correct minor typos
+that don't significantly impair meaning and understanding.
+
+If you're unsure, in addition to the general guidance above,
+we encourage you to reach out first before opening a PR here.
+For example, you could contact the PEP author(s),
+propose your idea in a discussion venue appropriate to the PEP (e.g.
+`Typing-SIG <https://mail.python.org/archives/list/typing-sig@python.org/>`__
+for static typing, or the `Packaging Discourse
+<https://discuss.python.org/c/packaging/>`__ for packaging), and/or
+open an `issue on this repo <https://github.com/python/peps/issues>`__.
+
+
 Commit messages and PR titles
 -----------------------------
 
 When adding or modifying a PEP, please always include the PEP number in the
 commit summary and pull request title.
 For example, ``PEP NNN: <summary of changes>``.
+Likewise, prefix rendering infrastructure changes with ``Infra:``,
```

Comment:
I think we can skip this sentence for brevity. It's not that important that the prefixes are consistent (though I agree with you that we should avoid obscure abbreviations like "PRS").

Reply:
We can, though I don't think it hurts to be provide some basic guidance here, since a lack of such was how we got cryptic ad-hoc initialisms like "PRS" in the first place, and its helpful to see at a glance the general high-level topic of a commit/PR without having to parse the full title/commit message.

Reply:
We could also drop the "Lint" prefix and just have "Infra" and "Meta", if that would simplify things.

Reply:
"Lint", "Infra" and "Meta" are good.

I'm also confused about "PRS", what does it stand for? It's very difficult to search for :)

Reply:
Seems like opinions were mixed about specifying particular prefixes (@hugovk and me :+1: , @JelleZijlstra and @AA-Turner :+1: ), but us three plus Guido all found "PRS" very confusing, so I dropped this sentence for now on the condition that we all avoid bespoke initialisms like that going forward. Deal?

> I'm also confused about "PRS", what does it stand for? It's very difficult to search for :)

AFAIK @AA-Turner made it up, apparently (per his responses to Guido, Jelle, and me who all asked the same question). I believe it was intended to stand for "PEP Rendering System".